### PR TITLE
Add the 1.2.9 Release  Note

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Add the 1.2.9 release note.
> Fixes #

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: downloads/release-notes/1.2.9/RELEASE_NOTE.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+If you have not installed jBallerina, then download the [installers](/downloads/) to install.
+
+#### Standard Library
+
+##### Transactions
+
+Improved logging for transactions. Redundant info logs were changed to debug logs.
+
+#### HTTP
```
Comment (chamil321) [inline downloads/release-notes/1.2.9/RELEASE_NOTE.md:32]:
```suggestion
##### HTTP
```

File: downloads/release-notes/1.2.9/RELEASE_NOTE.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+- Upgraded the Netty framework version to 4.1.50 and upgraded the SnakeYAML version to 1.26.
+- Added support for disabling the SSL certificate validation for HTTP2 clients.
+
+#### Security
```
Comment (chamil321) [inline downloads/release-notes/1.2.9/RELEASE_NOTE.md:37]:
```suggestion
##### Security
```

File: downloads/release-notes/1.2.9/RELEASE_NOTE.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+Added support to read custom/optional fields from the OAuth2 introspection response.
+
+#### Cache
```
Comment (chamil321) [inline downloads/release-notes/1.2.9/RELEASE_NOTE.md:41]:
```suggestion
##### Cache
```

File: downloads/release-notes/1.2.9/RELEASE_NOTE.md (hunk 1 above)
Comment (chamil321) [reply downloads/release-notes/1.2.9/RELEASE_NOTE.md:32]: Transactions, HTTP, Security, and Cache comes under Standard Library section.

**Review by keizer619 — APPROVED** [review]

**Pull request merged**